$NetBSD: patch-aw,v 1.2 2012/09/12 14:12:29 dsainty Exp $

Expose glXGetProcAddress and glXGetProcAddressARB.

The consequences of not making this patch are platform dependent - some build
chains will still expose the symbols, but some will not.  The API expects
them to be exposed.

--- src/mesa/drivers/x11/glxapi.c.orig	2007-11-01 16:30:52.000000000 +0100
+++ src/mesa/drivers/x11/glxapi.c
@@ -1391,6 +1391,7 @@
  * This function does not get dispatched through the dispatch table
  * since it's really a "meta" function.
  */
+PUBLIC
 __GLXextFuncPtr
 glXGetProcAddressARB(const GLubyte *procName)
 {
@@ -1407,6 +1408,7 @@
 
 
 /* GLX 1.4 */
+PUBLIC
 void (*glXGetProcAddress(const GLubyte *procName))()
 {
    return glXGetProcAddressARB(procName);
